firesteel/src/main/java/com/hp/hpl/firesteel/shuffle/ShuffleStoreManager.java

```diff
@@ -64,8 +64,8 @@ private ShuffleStoreManager () {
      */
     private void initNativeLibrary(String libraryName) {
          try {
-             System.loadLibrary(System.mapLibraryName(libraryName));
-             LOG.info(libraryName + " shared library loaded via mapLibraryName");
+             System.loadLibrary(libraryName);
+             LOG.info(libraryName + " shared library loaded via System.loadLibrary");
```

Comment:
Changes are fine.  Since "loadLibrary" throws 2 more exceptions ( Security-related and NULL name ), can you catch them @ line 69.

Reply:
@venkyn2 
For sure! I have implemented additional 2 exception handling.
Could you review my PR again?